### PR TITLE
Resolves switched out of realtime warning

### PR DESCRIPTION
**Pull request by madan96**
The controller_manager got stuck when loading rewd_controllers and system throws the warning
'Switched out of RealTime'. Looking at `var/log/syslog`, it appears to switch out when calling the logger.log() method. Commenting out the `logger.log()` calls seems to fix the issue. But it still remains a mystery as to why this is happening.

A possible issue could be updates in system-level libraries. 
@egordon : The only thing the logger does is push to an `std::queue`, so it is possible that `std::queue` is no longer real-time safe. A possible solution could be to use `boost::lockfree::queue`

### CODE REVIEW

**Comment by brianhou** [comment]
Thanks for finding this! I think we should just strip out the logger entirely: it's something that we added while attempting to debug the WAMs, and it looks like we weren't especially careful. (We only kept it because it didn't seem to cause problems.)

If we decide to implement it again, we should probably just figure out how to use `barrett::log::RealTimeWriter`.

**Review by brianhou — CHANGES_REQUESTED** [review]
I think we should strip out the logger entirely.

**Comment by madan96** [comment]
Oops! Seems like @aditya-vk configured the user details on `herb0`. Anyway, I have removed the logger and tested the f/t controller after rebuilding `libbarrett`. Works fine!

File: src/bus/can_socket-rtcan.cpp [hunk 1]
```diff
@@ -64,21 +64,20 @@ struct can_handle {
 };
 }
 
-double printCycle = 0.1;
 double startTime = highResolutionSystemTime();
 // std::stringstream sstm;
 // sstm << "/home/robot/log_" << startTime << ".log";
 std::string filename = "/home/robot/log_" + boost::lexical_cast<std::string>(startTime);;
```
Comment (brianhou) [inline src/bus/can_socket-rtcan.cpp]: I don't think we need these variables here (we can delete the logging and stack trace stuff from the constructor too).

**Review by brianhou — APPROVED** [review]

**Comment by brianhou** [comment]
Thanks! Merge whenever you're satisfied.

**Pull request merged**